### PR TITLE
Fix XLIFF import

### PR DESCRIPTION
**Pull request by timoludwig**
### Short description

This fixes the XLIFF import

### Proposed changes

- Add tests for xliff import/export
- Fix change detection for XLIFF import
- Allow importing unchanged XLIFF files

### Resolved issues

Fixes: #1276
Fixes: #1277

### CODE REVIEW

File: integreat_cms/xliff/utils.py [hunk 1]
```diff
@@ -375,6 +398,9 @@ def get_xliff_import_errors(request, page_translation):
     :param page_translation: The page translation which is being imported
     :type page_translation: ~integreat_cms.cms.models.pages.page_translation.PageTranslation
 
+    :param add_message_if_unchanged: Whether a message should added if  no changes are detected
```
Comment (svenseeberg) [inline integreat_cms/xliff/utils.py:401]:
```suggestion
    :param add_message_if_unchanged: Whether a message should added if no changes are detected
```

File: tests/xliff/xliff_test.py [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+        # If the role should be allowed to access the view, we expect a successful result
+        assert response.status_code == 302
+        page_tree = reverse(
+            "pages", kwargs={"region_slug": "augsburg", "language_slug": "en"}
+        )
+        assert response.headers.get("Location") == page_tree
+        response = client.get(page_tree)
+        print(response.headers)
+        # The download link to the xliff file is contained in the message
+        match = re.search(
```
Comment (svenseeberg) [inline tests/xliff/xliff_test.py:47]: I'd prefer a real XML parser in such cases, but I do not object a regex for this limited scope.

**Review by svenseeberg — APPROVED** [review]
As always just some comments, no requests for changes. Awesome tests :rocket:

File: tests/xliff/xliff_test.py (hunk 2 above)
Comment (timoludwig) [reply tests/xliff/xliff_test.py]: Good idea, I replaced the regex with:
```
parsed_content = fromstring(response.content.decode("utf-8"))
download_links = [
    link.attrib["href"]
    for link in parsed_content.iter("a")
    if "data-auto-download" in link.attrib
]
```

**Pull request merged**